cves/kernel/CVE-2016-4578.yml

```diff
@@ -274,8 +282,11 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
```

Comment:
Try to avoid the double contraction "shouldnt've", and break this statement down into a two sentences to improve readability. Add punctuation.